pkg/cfn/manager/tasks.go

```diff
@@ -178,19 +178,20 @@ func (t *asyncTaskWithStackSpec) Do(errs chan error) error {
 	return err
 }
 
-func doSingleTask(allErrs chan error, task Task) {
+func doSingleTask(allErrs chan error, task Task) bool {
 	desc := task.Describe()
 	logger.Debug("started task: %s", desc)
 	errs := make(chan error)
 	if err := task.Do(errs); err != nil {
 		allErrs <- err
-		return
+		return false
```

Comment:
Why not return an error instead?

Reply:
Because errors are handled via the channel already and we don't need to worry about the content as it's already take care of by the other mechanism, so this is just a hint, and boolean is easier to check.